match/new_orleans_so.py

```diff
@@ -89,28 +69,7 @@ def deduplicate_cprr_20_personnel(cprr):
     )
     clusters = matcher.get_index_clusters_within_thresholds(decision)
     # canonicalize name and uid
-    for cluster in clusters:
-        uid, first_name, last_name, middle_name = None, "", "", ""
-        for idx in cluster:
-            row = df.loc[idx]
-            if (
-                uid is None
-                or len(row.first_name) > len(first_name)
-                or (
-                    len(row.first_name) == len(first_name)
-                    and len(row.last_name) > len(last_name)
-                )
-            ):
-                uid, first_name, last_name, middle_name = (
-                    idx,
-                    row.first_name,
-                    row.last_name,
-                    row.middle_name,
-                )
-        cprr.loc[cprr.uid.isin(cluster), "uid"] = uid
-        cprr.loc[cprr.uid == uid, "first_name"] = first_name
-        cprr.loc[cprr.uid == uid, "last_name"] = last_name
-        cprr.loc[cprr.uid == uid, "middle_name"] = middle_name
```

Comment:
There really is a case with`middle_name` column involved. We could go even one step further, allowing the name columns be specified in a list e.g. `canonicalize_officers(cprr, id_column='uid', extra_columns=['first_name', 'last_name', 'middle_name'])`

Reply:
The changes that I just pushed seem to allow for the function to work, regardless of a` middle_name `column, without a hitch.

Reply:
Not critical. We can modify it when we need to in the future.